features/support/capybara.rb

```diff
@@ -95,29 +95,28 @@ def is_a?(klass)
 end
 
 Capybara.register_driver :headless_chrome do |app|
-  options = Selenium::WebDriver::Chrome::Options.new
+  options = Selenium::WebDriver::Options.chrome(
+    logging_prefs: { performance: 'ALL', browser: 'ALL' }
+  )
 
-  options.add_argument('--headless')
+  options.add_argument('--headless=new')
   options.add_argument('--no-sandbox')
   options.add_argument('--disable-popup-blocking')
   options.add_argument('--window-size=1280,2048')
   options.add_argument('--host-resolver-rules=MAP * ~NOTFOUND , EXCLUDE *localhost*')
   options.add_argument('--disable-search-engine-choice-screen')
+  options.add_argument('--disable-gpu')
 
   options.add_preference(:browser, set_download_behavior: { behavior: 'allow' })
-  options.add_option(:w3c, false)
-  options.add_option(:perfLoggingPrefs, { enableNetwork: true })
-  caps = Selenium::WebDriver::Remote::Capabilities.chrome(
-    loggingPrefs: { performance: 'ALL', browser: 'ALL' }
-  )
+
+  options.add_option(:perf_logging_prefs, { enableNetwork: true })
```

Comment:
These are due to changes in `selenium-webdriver` API. Not sure if it works properly, or how to test it. I guess, we'll figure it out if we need it :grimacing:

Reply:
Can't this be added to `logging_prefs` above?

To test see https://github.com/3scale/porta/blob/c0ec8861d5f44a160b13a94317e092e055156c4e/features/support/hooks.rb#L121-L135

On a web scenario failure, some data from the browser is being collected.

Reply:
ah, OK, thanks! that actually fails
I'll try to fix it